src/internal/NeoEsp32I2sXMethod.h

```diff
@@ -241,27 +240,52 @@ class NeoEsp32I2s0Mux8Bus
         //  encode bit #   0        1        2        3
         //  value zero     1        0        0        0
         //  value one      1        1        1        0    
-        const uint32_t EncodedZeroBit = 0x80000000;
-        const uint32_t EncodedOneBit = 0x80808000;
+        const uint32_t EncodedZeroBit = 0x00800000;
```

Comment:
add a comment, due to indianess between peripheral and cpu, bytes within the words are swapped in the const

Reply:
OK, the comment was added